core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceInputCheckerServiceSpec.js

```diff
@@ -18,19 +18,82 @@
 
  describe('Multiple Choice Input Checker Service', function() {
    var multipleChoiceInputCheckerService;
+   var stubs ;
+   var validInput ;
+   var invalidInput;
 
-   beforeEach(module('oppia'));
+   beforeEach(function() {
+     validInput = {
+       answerGroups = [
+         {
+           ruleSpecs : [
+             {
+               inputs : {
+                 x : 1;
+               },
+               rule_type : "Equals";
+             }
+           ],
+           outcome : {
+             dest : "Question 1";
+           }
+         }
+       ],
+       customizationArgs = {
+         choices : {
+           value : [
+             "<p>Option 1</p>"
+           ]
+         }
+       }
+     };
+
+     invalidInput  = {
```

Comment:
IMO a comment should be added to clarify what has made this sample, invalid sample as their can be multiple ways to be invalid!. Also please compact the code below with `[{`